core/src/test/scala/integration/kafka/api/ConsumerBounceTest.scala

```diff
@@ -77,8 +83,9 @@ class ConsumerBounceTest extends AbstractConsumerTest with Logging {
     }
   }
 
-  @Test
-  def testConsumptionWithBrokerFailures(): Unit = consumeWithBrokerFailures(10)
+  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumName)
+  @ValueSource(strings = Array("zk", "kraft", "kraft+kip848"))
+  def testConsumptionWithBrokerFailures(quorum: String): Unit = consumeWithBrokerFailures(10)
```

Comment:
This test is very flaky with kraft and kraft+kip848

Reply:
Setting `broker.session.timeout` and `max.poll.interval` to be 30000ms makes the test pass.

Reply:
Moved the test to [a new pr](https://github.com/apache/kafka/pull/15547) to unblock this one. Need to fix it later.